Parent/Parent/Router/Router+Errors.swift

```diff
@@ -1,104 +0,0 @@
-//
-// This file is part of Canvas.
-// Copyright (C) 2016-present  Instructure, Inc.
-//
-// This program is free software: you can redistribute it and/or modify
-// it under the terms of the GNU Affero General Public License as
-// published by the Free Software Foundation, either version 3 of the
-// License, or (at your option) any later version.
-//
-// This program is distributed in the hope that it will be useful,
-// but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-// GNU Affero General Public License for more details.
-//
-// You should have received a copy of the GNU Affero General Public License
-// along with this program.  If not, see <https://www.gnu.org/licenses/>.
-//
-
-import Foundation
-
-
-import Marshal
-import CanvasCore
-
-extension Router {
-    func defaultErrorHandler() -> ((UIViewController, NSError) -> ()) {
```

Comment:
rather than preserve this error alerting logic, I opted to just use the `ErrorReporter.reportError` which already does a bunch to present network errors with appropriate detail.